### PR TITLE
[Design Configurator] Add Name, Namespace, Annotations and Labels in the RJSF form in Meshery UI

### PR DESCRIPTION
**Pull request by Abhishek-kumar09**
**Description**

This PR fixes #
Screenshot 2022-10-03 at 5 25 33 PM

**Notes for Reviewers**
Annotations and Labels are set to root level properties in the pattern-yaml as a sibling to name and namespace, does it follow the current struct, @Revolyssup ?

**[Signed commits](https://github.com/meshery/meshery/blob/master/CONTRIBUTING.md#signing-off-on-commits-developer-certificate-of-origin)**
- [ ] Yes, I signed my commits.

### CODE REVIEW

File: ui/components/MesheryMeshInterface/PatternServiceForm.js [hunk 1]
```diff
@@ -76,6 +75,32 @@ function PatternServiceForm({ formData, schemaSet, onSubmit, onDelete, reference
...
+      type : "string"
+    };
+    schemaSet.workload.properties.annotation = {
+      description : "The annotation for the service",
+      default : "",
+      type : "string"
+    };
+  }, [])
+
+  console.log({ formData, schemaSet, onSubmit, onDelete, reference, namespace, onSettingsChange, onTraitsChange, scroll })
```
Comment (Abhishek-kumar09) [inline ui/components/MesheryMeshInterface/PatternServiceForm.js:102]:
```suggestion
```

**Review by leecalcote — APPROVED** [review]
The design looks good.

**Comment by leecalcote** [comment]
We do need a different shade for the initial background color.

**Comment by harkiratsm** [comment]
Is it the desired behaviour to have it like this ? // @leecalcote @Abhishek-kumar09 

image

**Comment by Revolyssup** [comment]
@Abhishek-kumar09  labels and annotations should be key-value pairs.

**Comment by leecalcote** [comment]
Feedback pending response.

**Comment by Abhishek-kumar09** [comment]
New Updates from the previous commit: 
1. The labels and annotations are in the form of additonalProperties, so they will be asking for "key" and "value"
2. The default assignments to the rjsf properties has been removed, that means there will not be any default assignment from the frontend and thus, the automatic additonal of empty keys in the yaml won't be there, just those properties which user tries to change will be there in the YAML string.
3. The description of additonal properties were not working fine before, that is fixed now.

**Comment by leecalcote** [comment]
@harkiratsm please review.

**Comment by leecalcote** [comment]
Until the upgrade to rjsfv5, users should still see "newKey", right?

Screen Shot 2022-10-07 at 4 27 38 PM

**Pull request merged**